report/renderer/report-renderer.js

```diff
@@ -227,13 +227,14 @@ export class ReportRenderer {
       headerContainer.classList.add('lh-header--solo-category');
     }
 
+    // Can't work with the fragment as we may move the element later
+    const scoreScale = this._dom.createComponent('scorescale').firstElementChild;
```

Comment:
latent bug, if we ever added a second element here (like a style sheet).

can we wrap the fragment in a newly create element instead?